### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by dungpa**
Spell-checked all the things :-).

### CODE REVIEW

File: src/FSharpVSPowerTools.Logic/VSUtils.fs [hunk 1]
```diff
@@ -318,7 +317,7 @@ type DocumentEventListener (events: IEvent<unit> list, delayMillis: uint16, upda
                                 Interval = TimeSpan.FromMilliseconds (float delayMillis))
     let tokenSource = new CancellationTokenSource()
 
-    // This is none or forall option for unit testing purpose only
+    // This is none or for-all option for unit testing purpose only
```
Comment (OkayX6) [inline src/FSharpVSPowerTools.Logic/VSUtils.fs]: *a* none or for-all.. ?

File: tests/FSharpVSPowerTools.Tests/SyntaxConstructClassifierTests.fs [hunk 2]
```diff
@@ -123,7 +123,7 @@ let internal f() = ()
         helper.SetUpProjectAndCurrentDocument(VirtualProjectProvider(buffer, fileName), fileName)
         let classifier = helper.GetClassifier(buffer)
 
-        // first event is raised when "fast calculatable" spans (without Unused declarations and opens) are ready
+        // first event is raised when "fast calculable" spans (without Unused declarations and opens) are ready
```
Comment (OkayX6) [inline tests/FSharpVSPowerTools.Tests/SyntaxConstructClassifierTests.fs]: Would you rather say "computable"?

**Comment by OkayX6** [comment]
Awesome! :)

**Comment by dungpa** [comment]
Done.

**Comment by vasily-kirichenko** [comment]
Great, I don't know English as well to review this PR though :)

**Pull request merged**